src/utils/transforms.lisp

```diff
@@ -500,6 +503,9 @@
   "Convert a DB3 logical value to a PostgreSQL boolean."
   (if (member value '("?" " ") :test #'string=) nil value))
 
+(defun progress-logical-to-boolean (value)
+    (if (string= value "yes") "1" (if (string= value "no") "0" nil))
```

Comment:
A more lispy way to write the same thing, easier to maintain and read, is the following:

``` lisp
(cond ((string-equal value "yes") "1")
      ((string-equal value "no")  "0")
      (t nil))
```